.circleci/config.yml

```diff
@@ -1,50 +1,79 @@
 ---
-version: 2
+version: 2.1
+
+executors:
+  node:
+    docker: [{image: 'circleci/node:10'}]
+
+  ruby:
+    docker: [{image: 'circleci/ruby:2.6.1'}]
+
+commands:
+  danger:
+    description: 'Run Danger'
+    steps:
+      - run:
+          command: if ! [[ $CIRCLE_PR_NUMBER ]]; then yarn run danger --id $task; fi
+          when: always
+          environment:
+            DANGER_DISABLE_TRANSPILATION: 'true'
+
+  embed-env-vars:
+    description: 'Embed environment variables into the app'
+    steps:
+      - run: touch .env.js
+
+  restore-cache-bundler:
+    description: 'Restore the Bundler cache'
+    steps:
+      - restore_cache:
+          key: 'v2-ruby-dependencies-{{ arch }}-{{ checksum "Gemfile.lock" }}'
```

Comment:
These giant strings should use YAML references, I think, so we don't wind up (say) changing `save` to v4 and forgetting `restore` at v3. We can just annotate this first string like `key: &bundler-cache-key 'v2-…'`, and then use it in the `save` command as `*bundler-cache-key`.

IMO anyway.

Reply:
That's out of scope for this PR, but a good idea.